src/views/SchemaForm.vue

```diff
@@ -173,6 +175,13 @@ export default {
             })
             instance.$mount()
             return this.$refs.container.appendChild(instance.$el)
+          } else if (isBoolean) {
```

Comment:
If you know how to do this with fewer lines

Reply:
I don't know if it answers your question, but I think the following code could be factorized in a function that takes a Field as parameter.

```
             const RadioFieldClass = Vue.extend(RadioField)
             let instance = new RadioFieldClass({
                 propsData: {field: field}
             })
             instance.$mount()
             return this.$refs.container.appendChild(instance.$el)
```